### ISSUE TITLE
Loading b.html on linux doesn't complete

### ISSUE THREAD

**Issue by fabricedesre** [servo/servo#11956]
I'm running current master (87d08eb4e8282f591261250d6b1d061fbb0584b6) on Ubuntu 16.04

Running with ./mach run -r -b we fail to fully load. The progress bar stops at ~ 10%, and no icon show up in the grid (the labels are here though).

Loading pages directly (eg. ./mach run -r https://github.com/) works.

**Comment by paulrouget** [comment]
Do you mind re-testing? There was a browserhtml update right after that commit.

**Comment by fabricedesre** [comment]
Tested again with 2ff6c3526313b9c0467e26f0236ae2578b7e57c5 but it's not better.

All events seems to be stuck. I can't close the main window cleanly.

**Comment by fabricedesre** [comment]
I clobbered my build: a debug build works, while a release one gets stuck.

**Comment by asajeffrey** [comment]
Confirmed, including with the current bhtml.

**Comment by asajeffrey** [comment]
IRC chat: http://logs.glob.uno/?c=mozilla%23servo&s=30+Jun+2016&e=30+Jun+2016#c467653

TL;DR: bisecting release builds, what fun. 90e970a6d2c0f3eebbff23edd2aa74f3a201e6d8 is the most likely candidate.

**Comment by asajeffrey** [comment]
Testing from 8052f5b80d812a3d266a03378ac610b5b544ef72 doesn't give the problem, so it looks like we have a culprit.

**Comment by asajeffrey** [comment]
Fixed by https://github.com/asajeffrey/servo/tree/revert-11949, which reverts #11949.

**Comment by asajeffrey** [comment]
@glennw is bisecting webrender to see if we can find out which WR change caused this. We can always apply #11968, but it would be nice not to keep most of the WR changes that came with #11949.

**Comment by glennw** [comment]
Fixed by https://github.com/servo/webrender/pull/306
